fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/load/ChunkWriter.java

```diff
@@ -286,4 +263,33 @@ private void pushImportOperationOutcomes2COS(ImportTransientUserData chunkData)
             chunkData.getBufferStreamForImportError().reset();
         }
     }
+    
+    @Override
+    public void open(Serializable checkpoint) throws Exception {
+        if (fhirValidation != null) {
+            isValidationOn = fhirValidation.equalsIgnoreCase("Y");
```

Comment:
this is fixed, or configurable?

Reply:
this is configurable through job parameter, if without setting in job parameter, then the default value false is used at present. 
I don't think we need to add it to the job parameter of the job operation, I think we can simply change the default value to true directly if need instead of adding a new config.